junifer/api/functions.py

```diff
@@ -239,15 +239,15 @@ def queue(
         if the ``jobdir`` exists and ``overwrite = False``.
 
     """
-    valid_kind = ["HTCondor"]
+    valid_kind = ["HTCondor", "GNUParallelLocal"]
     if kind not in valid_kind:
         raise_error(
             f"Invalid value for `kind`: {kind}, "
             f"must be one of {valid_kind}"
         )
 
     # Create a folder within the CWD to store the job files / config
-    jobdir = Path.cwd() / "junifer_jobs" / jobname
+    jobdir = Path.cwd() / "junifer_jobs" / jobname / kind.lower()
```

Comment:
If an user tries out different queueing system but with the same job name (which will most likely be the case), then this is the simplest way imo to handle the assets for each system.

Reply:
From what I know, the junifer queue command does not allow to use an existing `jobdir`.

Reply:
I don't quite understand how that can be a problem as the `jobdir` includes the queue kind dir.

Reply:
Updating junifer creates an inconsistent `junifer_jobs` directory.

Reply:
We can add it to the release notes and / or issue a note in the documentation. Do you have an alternative for this?

Reply:
My take is that we don't need the `kind.lower()` part, as you won't be able to run `junifer queue` if the `jobname` directory exists. It will be, technically, a different job with a different name.

I don't see any usecase to complicate ourselves like this.

Reply:
The user needs to change the YAML anyways (the `queue` section). 

Usually the testing is done with `junifer run`. You don't need to test the _queueing mechanism_ unless you are doing some advanced things. If you want to test the non-interactive run, even the GNU parallel tool is not the way. You should just run `junifer queue` with `--element` to queue only one element.

Reply:
Fair enough.